chapters/bizleg-case-studies/startups/yellr.md

```diff
@@ -0,0 +1,208 @@
+
+Security: Making sure we have preventative software as well as hiring security firms to find backdoors into our systems will be essential in protecting customer credit card numbers and location
+
+## R&D Objectives
+
+How will our product stay up to date?
+
+Since Yellr is open source, we will continually build in features as asked by the community and leverage an volunteer community of contributors. Customers of the Yellr Company can contract the Yellr Company to build features for them.
+
+Having an active volunteer contributor community will allow us to loss-lead out competitors in terms of feature development at a much smaller price. The use of the AGPLv3 and other copyleft software licenses protects our assets from being pulled into proprietary software projects and therefore covers any issues of people using our code as a derivative work.
+
+Red Hat has shown to be a great success at using their customers as their contributors. Their support model has made it very easy for engineers to implement small features themselves that would take too long for Red Hat to implement and also be convenient. A similar model can be taken with Yellr so that we provide a comprehensive support and hosting package that makes using our software as easy as possible while also allowing our customers to have access to the source code that we deploy. 
+
+## Action Plan
+
+### Product
+
+*What is our product? How will it saturate the market?*
+
+Our product will saturate the market mainly through the engagement with individuals/groups within the community and eventually towards our target market of news organizations and professional journalists.
+
+#### Key Features:
+*What makes it different from the competition?*
+            *see market share*
+
+*Price*
+Yellr will be priced on a tiered pricing system:
+
+| Feature                              | Bronze | Silver | Gold  |
+| -------                              | ------ | ------ | ----- |
+| Local Feed                           | x      | x      | x     |
+| Collections                          | x      | x      | x     |
+| Local Assignments <br> (25x25 miles) | x      | x      | x     |
+| Direct Messages to Users             |        | x      | x     |
+| Unlimited Post Storage               |        | x      | x     |
+| Infinite Assignment Area             |        |        | x     |
+| 24/7 phone support                   |        |        | x     |
+
+## Promotion
+
+Yellr is currently partnered with WXXI, an NPR affiliate which has done the majority of the promotion so far for the app. We are also reaching out to local journalism programs at RIT to test the deployment of the app as a hyper-local reporting methodology. Once these programs are finished we will review and perform a post-mortem. The post-mortem will focus on the reports of journalism students who used the app to gather sourcing and media content from communities.
+
+## Positioning/Place:
```

Comment:
Woops. Yeah copy paste error. Do colons cause issues in markdown? Most markdown compilers I've used haven't had any trouble rendering the file to HTML